bpf/process/types/basic.h

```diff
@@ -1274,7 +1253,7 @@ filter_read_arg(void *ctx, int index, struct bpf_map_def *heap,
 		index++;
 		if (index > MAX_SELECTORS || !e->sel.active[index])
 			return filter_args_reject();
-		tail_call(ctx, tailcalls, index + 5);
+		tail_call(ctx, tailcalls, index + 6);
```

Comment:
I'm not so clear on why it's now +6 instead of +5 here. Perhaps a comment is warranted?